GenderPayGap.WebUI/BusinessLogic/Services/UpdateFromCompaniesHouseService.cs

```diff
@@ -18,59 +18,53 @@ public class UpdateFromCompaniesHouseService
-
-                CustomLogger.Debug($"Starting transaction - OrganisationId({organisationId})");
-                _DataRepository.BeginTransactionAsync(
-                        async () => {
-                            try
-                            {
-                                CustomLogger.Debug($"Updating SIC codes - OrganisationId({organisationId})");
-                                UpdateSicCode(organisation, organisationFromCompaniesHouse);
-
-                                CustomLogger.Debug($"Updating Address - OrganisationId({organisationId})");
-                                UpdateAddress(organisation, organisationFromCompaniesHouse);
-
-                                CustomLogger.Debug($"Updating Name - OrganisationId({organisationId})");
-                                UpdateName(organisation, organisationFromCompaniesHouse);
-
-                                CustomLogger.Debug($"Saving - OrganisationId({organisationId})");
-                                _DataRepository.SaveChangesAsync().Wait();
-                                _DataRepository.CommitTransaction();
-
-                                CustomLogger.Debug($"Saved - OrganisationId({organisationId})");
-                            }
-                            catch (Exception ex)
-                            {
-                                string message =
-                                    $"Update from Companies House: Failed to update database, organisation id = {organisationId}";
-                                CustomLogger.Error(message, ex);
-                                _DataRepository.RollbackTransaction();
-                            }
-                        })
-                    .Wait();
+                    companiesHouseApi.GetCompany(organisation.CompanyNumber);
+
+                try
+                {
+                    CustomLogger.Debug($"Updating SIC codes - OrganisationId({organisationId})");
+                    UpdateSicCode(organisation, organisationFromCompaniesHouse);
+
+                    CustomLogger.Debug($"Updating Address - OrganisationId({organisationId})");
+                    UpdateAddress(organisation, organisationFromCompaniesHouse);
+
+                    CustomLogger.Debug($"Updating Name - OrganisationId({organisationId})");
+                    UpdateName(organisation, organisationFromCompaniesHouse);
+
+                    CustomLogger.Debug($"Saving - OrganisationId({organisationId})");
+                    dataRepository.SaveChanges();
+
+                    CustomLogger.Debug($"Saved - OrganisationId({organisationId})");
+                }
+                catch (Exception ex)
+                {
+                    string message =
+                        $"Update from Companies House: Failed to update database, organisation id = {organisationId}";
+                    CustomLogger.Error(message, ex);
+                }
+

```

Comment:
We don't care about this being within a transaction (it's ok for just part of it to succeed).
I only put it in a transaction because I was copying a pattern from elsewhere.